### PR TITLE
ignore all pdf files

### PR DESCRIPTION
**Pull request by Many5900**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,4 @@
 .DS_Store
-.pdf
+*.pdf
 report/main.pdf
 report/styleguide.pdf
```
Comment (tbdlarsen) [inline .gitignore:4]:
```suggestion
```

Comment (tbdlarsen) [reply .gitignore:4]: these are invalid

**Review by tbdlarsen — CHANGES_REQUESTED** [review]

**Review by tbdlarsen — APPROVED** [review]
LOKGOTTOME

**Review by elmavukicevic — APPROVED** [review]
make twiter apology

**Review by glund23student — APPROVED** [review]

**Pull request merged**